deltacat/compute/compactor/model/compaction_session_audit_info.py

```diff
@@ -84,6 +84,13 @@ def records_deduped(self) -> int:
         """
         return self.get("recordsDeduped")
 
+    @property
+    def records_dropped(self) -> int:
```

Comment:
Should we explicitly name it to `records_deleted`? We have `records_deduped` are also dropped.

Reply:
Sounds like a good idea. Making the change